src/EnergyPlus/WeatherManager.cc

```diff
@@ -287,6 +296,241 @@ namespace WeatherManager {
 	// MODULE SUBROUTINES:
 
 	// Functions
+	void
+	clear_state()
+	{
+		Debugout = false ;
+		YearofSim = 1 ; // The Present year of Simulation.
+		EnvironmentReportNbr = 0 ; // Report number for the environment stamp
+		EnvironmentReportChr = ""; // Report number for the environment stamp (character -- for printing)
+		TimeStampReportNbr = 0; // Report number for the time stamp
+		TimeStampReportChr = ""; // Report number for the time stamp (character -- for printing)
+		WeatherDataReport = 0 ; // Report number for the weather data
+		WeatherFileExists = false ; // Set to true if a weather file exists
+		LocationTitle = ""; // Location Title from input File
+		LocationGathered = false; // flag to show if Location exists on Input File (we assume one is 
+
+		GetBranchInputOneTimeFlag = true ;
+		GetEnvironmentFirstCall = true ;
+		PrntEnvHeaders = true ; 
+		WeatherFileLatitude = 0.0 ;
+		WeatherFileLongitude = 0.0 ;
+		WeatherFileTimeZone = 0.0 ;
+		WeatherFileElevation = 0.0 ;
+		WeatherFileUnitNumber = 0 ; // File unit number for the weather file
+		for ( int i = 1; i <= 12; i++ )
```

Comment:
@EnergyArchmage Instead of a for loop, you can do this...

```cpp
GroundTemps = Array1D< Real64 >( 12, 18.0 );
GroundTempsFC = Array1D< Real64 >( 12, 0.0 );
```
and so on for all the variables you have in the two for loops.